### PR TITLE
Fix rolling avg/sum columns bug

### PR DESCRIPTION
**Pull request by orion-junkins**
Adding these columns would previously throw an error due to the first piece of the time series having NaNs.

An additional check to trim the y data accordingly is introduced. 

Usage of these columns is added into the run_grid_search_job script.

### CODE REVIEW

**Comment by orion-junkins** [comment]
@douglasdennis no longer a draft, sorry forgot to edit the header

File: environment.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: river-level-test-2
+name: river-level
```
Comment (orion-junkins) [inline environment.yml:1]: Changed this by mistake in last PR, fixing here

File: environment.yml [hunk 2]
```diff
@@ -13,6 +13,7 @@ dependencies:
   - pytest                  # Unit testing framework
   - boto3                   # AWS S3 interaction
   - s3fs                    # AWS S3 interaction TODO: Convert to only boto3 OR only s3fs (Issue #30)
+  - holidays=0.24           # Holiday calendar for time series forecasting
```
Comment (douglasdennis) [inline environment.yml:16]: I would add a comment here that you are pinning this due to a bug upstream. This is something where you'll forget why you did it.

**Review by douglasdennis — APPROVED** [review]
Looks good. 

Best practice for bug fixes is to add a failing test that turns green with your fix. That helps with documentation, regression, and reviews.

**Pull request merged**